go.mod

```diff
@@ -1,28 +1,32 @@
 module github.com/snyk/go-application-framework
 
-go 1.18
+go 1.21
+
+toolchain go1.22.0
```

Comment:
Issue: Watch out for this toolchain entry, this could/will cause issues in downstream projects.